src/Components/MinimalComponentModel.cs

```diff
@@ -193,6 +195,31 @@ public ProjectFolderItemType Type
         /// </summary>
         public bool CanTranslate { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether this component is followed.
+        /// </summary>
+        public bool IsFollowed { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this component is followed by current user.
+        /// </summary>
+        public bool IsFollowedByCurrentUser { get; set; }
+
+        /// <summary>
+        /// Gets or sets a list of users following the component.
+        /// </summary>
+        public List<MinimalUserModel> FollowedByUsers { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this component is shared.
+        /// </summary>
+        public bool IsShared { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of components referencing the component.
+        /// </summary>
+        public long SharedComponentsCount { get; set; }
```

Comment:
Is ReferencedComponentsCount a better naming?

Reply:
Shared is a term Deb/Paul came up with.  It means the component is being referenced by other components. this is a count of those components.

Reply:
Sure, if that's where we are going.